streaming/src/main/scala/org/apache/spark/streaming/ui/UIUtils.scala

```diff
@@ -186,4 +191,21 @@ private[streaming] object UIUtils {
       </td>
     }
   }
+
+  def decodeURLParameter(urlParam: String): String = {
+    var param = urlParam
```

Comment:
Is it necessary to add these method here?

Reply:
I mistakenly added these functions instead of using imports. Thanks.

Reply:
@shahidki31  all review comments fixed.